db/migrate/20200810053843_create_data_explorer_queries.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class CreateDataExplorerQueries < ActiveRecord::Migration[6.0]
+  def up
+    create_table :data_explorer_queries do |t|
```

Comment:
Does any of the columns need to be `null: false`?

Reply:
Only hidden, I will add it - reason is that empty rows to hide default system queries mentioned ⬆️